app/code/UpdateTask.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+class UpdateTask extends BuildTask
+{
+    /**
+     * @var string
+     */
+    protected $title = "Updates source markdown files";
+
+    /**
+     * @var string
+     */
+    protected $description = "Downloads and cleans source markdown documentation files";
+
+    /**
+     * @var bool
+     */
+    protected $enabled = true;
+
+    /**
+     * @var SS_HTTPRequest $request
+     *
+     * @todo test the initial unlink works with cloned modules
+     */
+    public function run($request)
+    {
+        $this->printLine("updating...");
+
+        $repositories = $this->getRepositories();
+
+        foreach ($repositories as $repository) {
+            $this->cloneRepository($repository);
+            $this->cleanRepository($repository);
+        }
+    }
+
+    /**
+     * @return string
+     *
+     * @todo document this new configuration parameter
+     */
+    private function getPath()
+    {
+        return ASSETS_PATH;
+    }
+
+    /**
+     * @param string $message
+     */
+    private function printLine($message)
+    {
+        $this->eol = Director::is_cli() ? PHP_EOL : "<br>";
```

Comment:
Why is this being cached if it is only ever used in the next line? There isn't even a local property defined for this...